backend/tracim_backend/models/context_models.py

```diff
@@ -581,6 +584,24 @@ def __init__(self, label: str, raw_content: str) -> None:
         self.raw_content = raw_content
 
 
+class LiveMessageQuery(object):
+    """
+    Folder Content update model
```

Comment:
This docstring seems to be a copy/paste?

Reply:
Right, i will solve this.
```suggestion
    Live Message query model
```